src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -58,7 +58,7 @@ public async Task TestGetFirstDiagnosticWithFixAsync()
         var project = workspace.CurrentSolution.Projects.Single().AddAnalyzerReference(reference);
         var document = project.Documents.Single();
         var unused = await fixService.GetMostSevereFixAsync(
-            document, TextSpan.FromBounds(0, 0), new DefaultCodeActionRequestPriorityProvider(), CancellationToken.None);
```

Comment:
the 'default' provider had no 'priority', which means 'run all analyzers, in all priority classes'

Reply:
I'm amenable to that idea.  THough i also like null + valid values to represent the same.  Given that priority was represented with null before, i'm ok sticking with this for now for simplicity

Reply:
It is slightly confusing that we have a `Default` priority (which doesn't run everything) and also `null`.  I think I'd also prefer something like a `RunAll` priority